src/current/cockroachcloud/cockroachdb-dedicated-on-azure.md

```diff
@@ -6,15 +6,11 @@ toc_not_nested: true
 docs_area: deploy
```

Comment:
Maybe you are already planning for this, but I think as this page is pared down we can probably move all this info to an Azure section of the cloud FAQS

Reply:
That's a good point, but I don't think we should do it quite yet, until a few more of the limitations are lifted.